contrib/configuration_scripts/WMIExporterStateConfiguration.ps1

```diff
@@ -0,0 +1,98 @@
+﻿cls
+$serviceName = "wmi_exporter"
+$MSMQCollector = "msmq"
+$IISCollector = "iis"
+$MSSQLCollector = "mssql"
+$MetricsPath = "C:\Metrics"
+$ImagePathlocation = "HKLM:\SYSTEM\CurrentControlSet\Services\wmi_exporter\"
+$RegString = "ImagePath"
+$ImagePathstr1 = '"C:\Program Files\wmi_exporter\wmi_exporter.exe" --log.format logger:eventlog?name=wmi_exporter --telemetry.addr :9182 --collector.textfile.directory C:\Metrics --collectors.enabled '
+$global:CoreCollectorsEnabled = "cpu,cs,logical_disk,net,os,process,service,textfile,system" -split ","
```

Comment:
You can use the `[defaults]` placeholder value here to not have to keep this list up to date as the defaults may change over time.